### DISCUSSION TITLE
the exception illegal instruction

### DISCUSSION THREAD

**Discussion by RUOKexe** [Alex313031/Thorium-Win#93] [Show and tell]
![Capture_2023_10_20_17_01_20_504](https://github.com/Alex313031/Thorium-Win/assets/148570728/e7045bf8-fde3-4178-9078-1985b6230d93)
when i click the thromium mini installer it said that the exception illegal instruction an attempt was made to execute an illegal instruction. 0xc000001d occurred in the application at location 0x00007ff62c0621c9. pls guys help me fix this i really want to use this browser -windows 10-

**Comment by gz83** [comment]
Mark as duplicate
